goad

```diff
@@ -52,8 +52,9 @@ else
 		git submodule update --init
 		;;
 	test)
-		go test -i "$SUBSECTION" &&
-		go test -v "$SUBSECTION" && {
+		set +e ; shift ; shift ; set -e
```

Comment:
`shift` fails if there's no remaining args.  The addition of the shifting and `"$@"`s is to allow tacking more args on, e.g. `goad test ./somepackage/... -race`, but `goad test` with no args is still valid.